hal/src/sercom/v2/pad/impl_pad_thumbv7em.rs

```diff
@@ -191,10 +215,11 @@ pad_table!(
         D: (Sercom5, Pad3, IoSet5, IoSet6),
     }
     PB02 {
-        D: (Sercom5, Pad0, IoSet6),
+        // According to Metro M4, PB02 is I2C-capable. This disagrees with datasheet table 6-8.
+        D: (Sercom5, Pad0, IoSet6) + I2C,
     }
-    PB03 {
-        D: (Sercom5, Pad1, IoSet6),
+    PB03 {// According to Metro M4, PB03 is I2C-capable. This disagrees with datasheet table 6-8.
```

Comment:
Is there somewhere we could document this? I don't have any immediate ideas, but it would be nice to keep track of which pins have undocumented functions.

Reply:
I documented it directly in the `sercom::v2` module-level documentation. Seems like a good place to put that stuff.